conftest.yaml

```diff
@@ -18,7 +18,7 @@ pipeline:
 
   - uses: go/build
     with:
-      ldflags: -w -s -X github.com/open-policy-agent/conftest/internal/commands.version=${{package.version}}
+      ldflags: -s -X github.com/open-policy-agent/conftest/internal/commands.version=${{package.version}}
```

Comment:
hmmmm i think we agree to not strip symbols, that's why we used to have '-s' before in default flags and then dropped. please drop '-s' too as we want symbols tables for govulnchecker to work.